File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-1.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int i = 0; int j = 1;
        int prev = nums[0];
        while (j < nums.size()){
            if (nums[j]!=prev){
                cout << nums[j] << endl;
                i++;
                nums[i] = nums[j];
                prev = nums[j];
            }
            j++;
        }
        return i+1;
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        m = {}
        i = 1
        count = 1
        maxcount = 0
        nums.sort()
        prev = nums[0]
        while i < len(nums):
            num = nums[i]
            if prev != num:
                if count in m:
                    m[count].append(prev)
                else:
                    m[count] = [prev]
                maxcount = max(maxcount, count)
                prev = num
                count = 1
            else:
                count += 1
            i+=1
        if count in m:
            m[count].append(prev)
        else:
            m[count] = [prev]
        maxcount = max(maxcount,count)
        ret = []
        while len(ret) <= k and maxcount > 0:
            if maxcount in m:
                for i in m[maxcount]:
                    ret.append(i)
                    if len(ret) == k:
                        return ret
            maxcount -= 1
        return ret